src/GitHub.App/Services/DialogService.cs

```diff
@@ -29,9 +29,13 @@ public class DialogService : IDialogService
             this.showDialog = showDialog;
         }
 
-        public async Task<CloneDialogResult> ShowCloneDialog(IConnection connection)
+        public async Task<CloneDialogResult> ShowCloneDialog(IConnection connection, string url)
```

Comment:
The interface adds a default value of `null` to this parameter. Should probably add it here as well to make the signatures match.